it/controllers/pensions/lifetimeAllowance/LifeTimeAllowanceAnotherWayControllerISpec.scala

```diff
@@ -0,0 +1,358 @@
+
+          lazy val result: WSResponse = {
+            dropPensionsDB()
+            authoriseAgentOrIndividual(user.isAgent)
+            insertCyaData(aPensionsUserData, aUserRequest)
+            urlPost(fullUrl(pensionLifeTimeAllowanceAnotherWayUrl(taxYearEOY)), body = form, follow = false, welsh = user.isWelsh,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY, validTaxYearList)))
+          }
+
+          "has the correct status" in {
+            result.status shouldBe BAD_REQUEST
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+          import Selectors._
+          import user.commonExpectedResults._
+          titleCheck(user.specificExpectedResults.get.expectedErrorTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          textOnPageCheck(expectedLumpSumText, paragraphSelector(3))
+          textOnPageCheck(user.specificExpectedResults.get.expectedParagraphText, paragraphSelector(4))
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          radioButtonCheck(yesText, 1, checked = Some(false))
+          radioButtonCheck(noText, 2, checked = Some(false))
+          buttonCheck(expectedButtonText, continueButtonSelector)
+          formPostLinkCheck(pensionLifeTimeAllowanceAnotherWayUrl(taxYearEOY), formSelector)
+          welshToggleCheck(user.isWelsh)
+
+          errorSummaryCheck(user.specificExpectedResults.get.expectedError, Selectors.yesSelector)
+          errorAboveElementCheck(user.specificExpectedResults.get.expectedError, Some("value"))
+        }
+      }
+    }
+
+    "redirect and update question to 'Yes' when there is currently no radio button value selected and the user selects 'Yes'" which {
+      lazy val form: Map[String, String] = Map(YesNoForm.yesNo -> YesNoForm.yes)
+
+      lazy val result: WSResponse = {
+        dropPensionsDB()
+        val pensionsViewModel = aPensionLifetimeAllowanceViewModel.copy(
+          pensionPaidAnotherWayQuestion = None
+        )
+
+        insertCyaData(pensionsUserDataWithLifetimeAllowance(pensionsViewModel), aUserRequest)
+
+        authoriseAgentOrIndividual(isAgent = false)
+        urlPost(fullUrl(pensionLifeTimeAllowanceAnotherWayUrl(taxYearEOY)), body = form, follow = false,
+          headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY, validTaxYearList)))
+      }
+
+      "has a SEE_OTHER(303) status and redirect Do you have a reduced annual allowance page" in {
+        result.status shouldBe SEE_OTHER
+        //TODO redirect page to "Your client's pension taken in another way" Page
+        result.header("location") shouldBe Some(pensionSummaryUrl(taxYearEOY))
+      }
+
+      "updates pensionPaidAnotherWayQuestion to Some(true)" in {
+        lazy val cyaModel = findCyaData(taxYearEOY, aUserRequest).get
+        cyaModel.pensions.pensionLifetimeAllowances.pensionPaidAnotherWayQuestion shouldBe Some(true)
```

Comment:
You have this test being checked 3 times in this block:
`cyaModel.pensions.pensionLifetimeAllowances.pensionPaidAnotherWayQuestion shouldBe`


Should be like this:

   ```
cyaModel.pensions.pensionLifetimeAllowances.pensionPaidAnotherWayQuestion shouldBe Some(true)
        cyaModel.pensions.pensionLifetimeAllowances.pensionPaidAnotherWay shouldBe aPensionLifetimeAllowanceViewModel.pensionPaidAnotherWay
        cyaModel.pensions.pensionLifetimeAllowances.aboveLifetimeAllowanceQuestion shouldBe
          aPensionLifetimeAllowanceViewModel.aboveLifetimeAllowanceQuestion
        cyaModel.pensions.pensionLifetimeAllowances.pensionAsLumpSumQuestion shouldBe aPensionLifetimeAllowanceViewModel.pensionAsLumpSumQuestion
        cyaModel.pensions.pensionLifetimeAllowances.pensionAsLumpSum shouldBe aPensionLifetimeAllowanceViewModel.pensionAsLumpSum
```

Reply:
Might as well add this too while you are doing it as is missing:
```
cyaModel.pensions.pensionLifetimeAllowances.aboveLifetimeAllowanceQuestion shouldBe
       aPensionLifetimeAllowanceViewModel.aboveLifetimeAllowanceQuestion
```